### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by ashly1yeo**
Removed outdated and broken links as well as update links to latest docs:
- removed preview sweepstakes link 
- updated photoaccess link to TOU to a direct link instead of outdated aka.ms link
- consolidate sendphotos and photoaccess pages for the single link on how to send photos

### CODE REVIEW

**Review by nairmanu — APPROVED** [review]

**Review by nairmanu — CHANGES_REQUESTED** [review]
Can you please take a look at the open comment?

File: faq.md [hunk 1]
```diff
@@ -55,7 +50,7 @@ If your photos are approved by the developer for use in their project, you will
 
 ***Android users***: You can download Trove from the **[Google Play Store](https://aka.ms/troveandroid)**
 
-***iOS users***: You can visit Trove's **[mobile website](https://aka.ms/troveweb)**
+***iOS users***: You can download Trove from the **[Apple App Store](https://apps.apple.com/us/app/trove-a-photo-marketplace/id1491532054)**
```
[the remark this answers is not in the document]
Comment (ashly1yeo) [reply faq.md:53]: Yeah I saw that short url but it is owned by Krishna and it has not been redirected yet.  I will check with Christian to see if he has access to change the destination url and update this link.

[the remark this answers is not in the document]
Comment (ashly1yeo) [reply faq.md]: link updated!

**Review by ashly1yeo — COMMENTED** [review]
@nairmanu link updated!

**Review by nairmanu — APPROVED** [review]

**Pull request merged**